api_examples.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+
+class APIExampleMixin:
+    """A TestCase mixin that allows access to API by Example retrieval methods."""
+
+    SERVER_NAME = 'localhost:8000'
+
+    def create_request(self, *args, **kwargs):
+        return super().create_request(*args, SERVER_NAME=self.SERVER_NAME, **kwargs)
+
+    def api_example_json_file(self, url, method):
+        """
+        Return the API example at a location generated from the view URL and method.
+
+        All URL kwargs in the URL are given only by their names, no other characters.
+        For example:
+            for POSTing to a view at `/patients/<pk>`,
+            call self.api_example_json_file('/patients/pk', 'post')
+            and you'll get 'api-description/patients/pk/post.json'
+        """
+        return '{base}/api-description{url}/{method}.json'.format(
+            base=self.EXAMPLES_DIR,
```

Comment:
That's true, although it'd be nice not to have to (unless we're already making this a `TestCase` for the other thing I commented on).